pkg/sfu/sfu.go

```diff
@@ -123,6 +124,7 @@ func NewWebRTCTransportConfig(c Config) WebRTCTransportConfig {
 	if len(c.WebRTC.Candidates.NAT1To1IPs) > 0 {
 		w.setting.SetNAT1To1IPs(c.WebRTC.Candidates.NAT1To1IPs, webrtc.ICECandidateTypeHost)
 	}
+	w.setting.SetICEMulticastDNSMode(ice.MulticastDNSModeDisabled)
```

Comment:
should we make this configurable?

Reply:
Not required in most scenarios
Disable mdns can save many file descriptors

Reply:
Would like to make this configurable if someone need :)

Reply:
made it configuabe

Reply:
@tarrencev @adwpc  There can be some security implication disabling the dns multicast for ice, so lets keep it enabled as default.

Reply:
OK